tox.ini

```diff
@@ -74,6 +74,8 @@ commands =
 
          tahoe --version
 
+	 python -c "import sys; print('sys.stdout.encoding:', sys.stdout.encoding)"
```

Comment:
Is it just github rendering or is this spaced/indented differently from the rest?

Reply:
Stray tab, thanks